src/store/bridge/mutations.js

```diff
@@ -25,3 +25,18 @@ export const setToAccount = (state, toAccount) => {
 export const setToNative = (state, flag) => {
   state.toNative = flag;
 };
+
+export const zeroToken = (state) => {
+  state.token = {
+    contract: "",
+    symbol: "Select a token",
+    balance: 0,
+    precision: 4,
+    enabled: true,
+    chain: "telos",
+    tportstart: true,
+    telosdio: false,
+    bridgestart: true,
+    amount: null,
```

Comment:
Is this the right way to do it?

Reply:
Yes, it's about right, but I think we can refactor as follows: Remove the `resetToken` action;  Rename the `zeroToken` mutation to `resetToken` and call the mutation directly using [commit](https://vuex.vuejs.org/guide/mutations.html#mutations-must-be-synchronous); Call the mutation directly because we aren't doing anything else in the function than resetting the token's state. I think the try-catch is also unnecessary in this case

Reply:
This will most likely be a quick fix :wink: